client/src/controllers/BulkController.test.js

```diff
@@ -0,0 +1,58 @@
+import { Application } from '@hotwired/stimulus';
+import { BulkController } from './BulkController';
+
+describe('BulkController', () => {
+  beforeEach(() => {
+    document.body.innerHTML = `
+    <div data-controller="w-bulk">
+      <input type="checkbox" data-w-bulk-target="all" data-action="w-bulk#toggleAll">
+      <div>
+        <input type="checkbox" data-w-bulk-target="item" disabled data-action="w-bulk#toggle">
+        <input type="checkbox" data-w-bulk-target="item" data-action="w-bulk#toggle">
+        <input type="checkbox" data-w-bulk-target="item" data-action="w-bulk#toggle">
+      </div>
+      <button data-action="w-bulk#toggleAll" data-w-bulk-force-param="false">Clear all</button>
+      <button data-action="w-bulk#toggleAll" data-w-bulk-force-param="true">Select all</button>
```

Comment:
```suggestion
      <button id="clear" data-action="w-bulk#toggleAll" data-w-bulk-force-param="false">Clear all</button>
      <button id="set" data-action="w-bulk#toggleAll" data-w-bulk-force-param="true">Select all</button>
```

Again, let's use IDs